src/core/Akka.Streams.TestKit.Tests/ScriptedTest.cs

```diff
@@ -200,7 +200,7 @@ public void Request(int demand)
 
             public bool ShakeIt()
             {
-                var oneMilli = TimeSpan.FromMilliseconds(1);
+                var oneMilli = TimeSpan.FromMilliseconds(10);
```

Comment:
So the issue, near as I can tell based on the additional debug logging I added: the `ReceiveWhile` never even had a chance to run. Bails out immediately in some small cases. I can't figure out what caused this to happen between .NET Core 1.1 and 2.1 but I'm not going to wait forever to get to the bottom of it.